src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/CompletionSource.cs

```diff
@@ -220,6 +220,7 @@ static void CheckForExperimentStatus(ITextView textView, Document document)
             SnapshotSpan applicableToSpan,
             CancellationToken cancellationToken)
         {
+            session.Properties[TriggerLocation] = triggerLocation;
```

Comment:
Why not do this in `GetCompletionContextWorkerAsync` ?

Reply:
Because we don't get the trigger location in `GetExpandedCompletionContextAsync` (which also calls `GetCompletionContextWorkerAsync`)